package.json

```diff
@@ -36,14 +36,14 @@
     "test:compatibility:debug": "node $NODE_DEBUG_OPTION node_modules/cucumber/bin/cucumber-js src/test/compatibility/features/ --require 'src/test/common/steps/*.js' --require 'src/test/compatibility/config' --format ./src/test/compatibility/custom-json-formatter.js:build/reports/compatibility-report.json",
     "test:compatibility:report": "node src/test/compatibility/compatibility-report.js",
     "compat:win": "cross-env BROWSER_STACK_OS=Windows BROWSER_STACK_OS_VERSION=10 npm-run-all -c -p --max-parallel 5 compat:win:ie compat:win:edge compat:win:chrome compat:win:firefox",
-    "compat:win:ie": "cross-env BROWSER_STACK_BROWSER=IE BROWSER_STACK_BROWSER_VERSION=11.0 cucumber-js src/test/compatibility/features/ --require 'src/test/common/steps/*.js' --require 'src/test/compatibility/config' --format ./src/test/compatibility/custom-json-formatter.js:build/reports/compatibility/win-ie-report.json",
+    "compat:win:ie": "cross-env BROWSER_STACK_BROWSER=IE cucumber-js src/test/compatibility/features/ --require 'src/test/common/steps/*.js' --require 'src/test/compatibility/config' --format ./src/test/compatibility/custom-json-formatter.js:build/reports/compatibility/win-ie-report.json",
     "compat:win:edge": "cross-env BROWSER_STACK_BROWSER=Edge BROWSER_STACK_BROWSER_VERSION=18.0 cucumber-js src/test/compatibility/features/ --require 'src/test/common/steps/*.js' --require 'src/test/compatibility/config' --format ./src/test/compatibility/custom-json-formatter.js:build/reports/compatibility/win-edge-report.json",
```

Comment:
There is currently a bug with BrowserStack where the latest version of Edge actually points to version 16.0 rather than 18.0 which I have raised with them, so I'll change this when that is fixed. In the meantime it was actually handy to make sure that works when we specify a mix of latest versions of browsers and specific versions.